### PR TITLE
Basic Portuguese Brazilian Translation

### PR DESCRIPTION
**Pull request by acneidert**
Not complete but most of the core interface

### CODE REVIEW

File: superset/config.py [hunk 1]
```diff
@@ -163,6 +163,7 @@
     'zh': {'flag': 'cn', 'name': 'Chinese'},
     'ja': {'flag': 'jp', 'name': 'Japanese'},
     'de': {'flag': 'de', 'name': 'German'},
+    'pt-BR': {'flag': 'pt-BR', 'name': 'Portuguese/Brazil'},
```
Comment (xrmx) [inline superset/config.py]: Looking at the other language, shouldn't this be `'pt': {flag: 'br', 'name': 'Brazilian portugese'}`?

Comment (xrmx) [inline superset/config.py]: Do you see the flag? Shouldn't this be something like?
```
    'pt-BR': {'flag': 'br', 'name': 'Brazilian Portuguese'}, 
```

Comment (AcNeidert) [reply superset/config.py]: It really is better, thank you!

**Comment by mistercrunch** [comment]
@betodealmeida

**Pull request merged**